templates/workloads/common/cf/envvars.yml

```diff
@@ -10,10 +10,10 @@ Environment:
   Value: !Sub '${EnvName}'
 - Name: COPILOT_SERVICE_NAME
   Value: !Sub '${WorkloadName}'
+{{- if .EnvControllerLambda}}
 - Name: COPILOT_LB_DNS
-  Value:
-    Fn::ImportValue:
-      !Sub "${AppName}-${EnvName}-PublicLoadBalancerDNS" {{if .Variables}}{{range $name, $value := .Variables}}
+  Value: !GetAtt EnvControllerAction.PublicLoadBalancerDNSName
+{{- end}}{{if .Variables}}{{range $name, $value := .Variables}}
```

Comment:
I'm slightly worried that we'll break things whenever we integrate the controller lambda with other services.
what do you think of doing this:
```
templates/workloads/common/cf/envvars.yml # only contains Environment, and common vars
templates/workloads/common/cf/secrets.yml # only contains Secrets
```
then in 
```yaml
# in templates/workloads/services/lb-web/cf.yml
{{include "envvars" . | indent 10}}
          - Name: COPILOT_LB_DNS
            Value: !GetAtt EnvControllerAction.PublicLoadBalancerDNSName
{{include "secrets" . | indent 10}}
```

Reply:
sure i'll do that!